packages/runtime/runtime-utils/src/index.ts

```diff
@@ -10,8 +10,8 @@ export * from "./objectstoragepartition";
 export * from "./objectstorageutils";
 export * from "./requestParser";
 export * from "./serializer";
-export * from "./summarizerNode";
-export * from "./summarizerNodeWithGc";
+// eslint-disable-next-line import/no-internal-modules
+export * from "./summarizerNode/index";
```

Comment:
This can just be `export * from "./summarizerNode";` right?

Reply:
I like the idea, just tried it and it seemed like it was going to work, but the build breaks.

Reply:
That's strange! I see this worked in a lot of places in the code - see `packages/framework/aqueduct` for instance. Curious as to why build breaks for you.